### PR TITLE
Add MS Teams support

### LINKED ISSUE #60: Add Microsoft Teams suppport
**Describe the solution you'd like**
We got a couple of requests for Microsoft Teams support at Kubernetes Days India. Need to explore possible integration.
https://products.office.com/en-US/microsoft-teams/group-chat-software

### PR DESCRIPTION
**Pull request by PrasadG193**
##### ISSUE TYPE

 - Feature Pull Request

##### SUMMARY

- Add support for Microsoft Teams
- Multicluster support not available yet for Teams

Integration tests will be addressed with a separate issue. Blocked by https://github.com/infracloudio/msbotbuilder-go/issues/46

Fixes #60

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -44,11 +46,11 @@ require (
...
-	gopkg.in/yaml.v2 v2.2.4
+	gopkg.in/yaml.v2 v2.2.8
 	k8s.io/api v0.17.0
 	k8s.io/apimachinery v0.17.0
 	k8s.io/client-go v0.17.0
 	k8s.io/kubectl v0.17.0
 )
 
-go 1.13
+go 1.12
```
Comment (PrasadG193) [inline go.mod]: Undo this

**Comment by PrasadG193** [comment]
image

**Comment by zxcoolman** [comment]
Nice job ! 
where do you find  'APPLICATION_ID' and 'APPLICATION_PASSWORD' ?
I would try this fonctionnality :)

Regards,

**Comment by PrasadG193** [comment]
Hey @zxcoolman,
We are still working on documentation Teams support. You can have a look at https://deploy-preview-45--objective-jones-39e308.netlify.app/installation/teams/ for steps to integration BotKube with Teams. Please let us know the feedback/suggestions if you have any.

**Comment by rutaveej-shah** [comment]
Hi 

Is it possible to specify the MsTeams webhook channel and get notification from that.
Thanks.

**Pull request merged**